main_chain.js

```diff
@@ -420,6 +420,38 @@ function updateMainChain(conn, from_unit, last_added_unit, onDone){
 		});
 	}
 
+	function findMinMcWitnessedLevel(tip_unit, min_level, arrWitnesses, handleMinMcWl){
```

Comment:
better use the same name first_unstable_mc_level instead on min_level

Reply:
ok